user_docs/en/changes.t2t

```diff
@@ -8,6 +8,7 @@ What's New in NVDA
 == New Features ==
 - Support for reporting notes in MS Excel with UI Automation enabled on Windows 11. (#12861)
 - In recent builds of Microsoft Word via UI Automation on Windows 11, the existence of bookmarks, draft comments and resolved comments are now reported in both speech and braille. (#12861)
+- The new --lang command line parameter allows to override the configured NVDA language. (#10044)

```

Comment:
The meaning is there but this line reads awkwardly.  I'd suggest either:
- The new --lang command line parameter allows you to override the configured NVDA language. (#10044) 
 or
- The new --lang command line parameter allows overriding the configured NVDA language. (#10044)

Reply:
@michaelDCurran  Given external contributors aren't really supposed to touch change log file would you be willing to modify the changes entry as per @Qchristensen's sugggestion when reviewing the latest commits and merging this PR? Aside from this it seems this is ready to go to master.